peter_rockwood_HW3/HW_3_peter_rockwood.js

```diff
@@ -0,0 +1,629 @@
+// ### Homework #3
```

Comment:
Looks good! Thanks for the concern about formatting. The `console.log`s in your week 4 homework are great, so I think we're on track!